gen3/lib/g3k_manifest.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+#
+# Little library of manifest related functions usually accessed
+# via the `g3k` cli defined in kube/kubes.sh
+#
+
+G3K_MANIFEST_SH_DIR=$(dirname "${BASH_SOURCE:-$0}")  # $0 supports zsh
+GEN3_HOME="${GEN3_HOME:-$(cd "${G3K_MANIFEST_SH_DIR}/../.." && pwd)}"
+GEN3_MANIFEST_HOME="${GEN3_MANIFEST_HOME:-"$(cd "${GEN3_HOME}/.." && pwd)/cdis-manifest"}"
+export GEN3_HOME
+export GEN3_MANIFEST_HOME
+
+#
+# Internal init helper
+#
+g3k_manifest_init() {
+  # do this at most once a day
```

Comment:
are you gonna use it for jenkins? do you want to support parameter to supply branch to use?

Reply:
That's just there to automatically checkout/update cdis-manifest/ if it's not already done.  Jenkins will ensure that it runs against the latest manifest independent of this `g3k_manifest_init` - it's just there so things work for a basic user ...